test/ShellWordsSpec.hs

```diff
@@ -39,27 +39,13 @@ testCases =
   , ("var --bar=baz\\ bat", ["var", "--bar=baz bat"])
   , ("var --bar baz\\ bat", ["var", "--bar", "baz bat"])
   , -- N.B. sh preserves escapes in quoted values, python-shellwords does not.
-    -- we behave like sh in this regard. See omitted cases below too.
+    -- we behave like sh in this regard.
     ("var --bar 'baz\\ bat'", ["var", "--bar", "baz\\ bat"])
   , ("var  --bar \"baz\\ bat\"", ["var", "--bar", "baz\\ bat"])
+  , ("var \"--bar=\\\\'baz\\\\'\"", ["var", "--bar=\\'baz\\'"])
+  , ("var --bar='\\\\'", ["var", "--bar=\\"])
```

Comment:
It might've just been my inability to correctly put the double-`\\` escapes in the test input that prevented me having these cases enabled before :shrug: